src/locomoset/metrics/classes.py

```diff
@@ -121,14 +121,16 @@ class MetricConfig(Config):
         model_name: Name of the HuggingFace model to perform metric experiment on.
         dataset_name: Name of the HuggingFace dataset to use for metric experiment.
         metrics: Which metrics to perform the experiments on.
+        save_dir: Where to save a local copy of the results.
+        dataset_args: Dict defining the splits and columns of the dataset to use, see
+            the docstring of the base Config class for details.
         run_name: Name of the run (used for wandb/local save location), defaults to
             {dataset_name}_{model_name}.
+        n_samples: How many samples to use in the metric experiments.
         random_state: Random state to use for train/test split.
         use_wandb: Whether to use wandb for logging.
         wandb_args: Arguments to pass to wandb.init.
```

Comment:
Minor comment, but is there a reason for calling this `wandb_args` on lower-level configs but `wandb` on top level configs? I have a preference for consistency across these argument names to minimise confusion